pppd/Makefile.am

```diff
@@ -0,0 +1,165 @@
+sbin_PROGRAMS = pppd
+man8_MANS = pppd.8 srp-entry.8
+if WITH_SRP
+sbin_PROGRAMS += srp-entry
+endif
+
+pkgconfigdir   = $(libdir)/pkgconfig
```

Comment:
`pkg.m4` already defines this

Reply:
I tried to remove this, and autoreconf barfed at me.